VGMPlay/VGMPlayUI.c

```diff
@@ -2339,7 +2384,7 @@ static void PlayVGM_UI(void)
 		if (! PausePlay && PlayingMode != 0x01)
 			WaveOutLinuxCallBack();
 		else
-			Sleep(100);
+			Sleep(50);
```

Comment:
curious to why this is changed

Reply:
Since dbus and vgmplay are doing work on the same thread, with vgmplay paused there's a noticeable delay from when you hit resume, to the moment it actually resumes.
This happens with all other actions that interact with dbus. This sleep is what causes it.

I switched it to 50 because the windows one that is a few lines above also uses the same value, and I also thought that it's still a reasonable value that wouldn't cause much more load while paused.

https://github.com/vgmrips/vgmplay/blob/e946c2d2b1d5699cf7a1c22e22c337b4dec53147/VGMPlay/VGMPlayUI.c#L2372-L2374